repotracker/repotracker.go

```diff
@@ -759,6 +760,15 @@ func createVersionItems(ctx context.Context, v *model.Version, ref *model.Projec
 				"version":         v.Id,
 			})
 			if err != nil {
+				cmdErr, isCmdErr := err.(mongo.CommandError)
+				if isCmdErr && cmdErr.HasErrorLabel(command.TransientTransactionError) {
+					grip.Notice(message.Fields{
+						"message":    "hit transient transaction error, will retry",
```

Comment:
is there a number of retries that we allow? should we track that?

Reply:
I feel like that's not too important since if it exceeds the max retries we log that separately